lib/matplotlib/backend_bases.py

```diff
@@ -1376,13 +1368,11 @@ class MouseButton(IntEnum):
 
 class MouseEvent(LocationEvent):
     """
-    A mouse event ('button_press_event',
-                   'button_release_event',
-                   'scroll_event',
-                   'motion_notify_event').
+    A mouse event ('button_press_event', 'button_release_event', \
+'scroll_event', 'motion_notify_event').
 
-    In addition to the `Event` and `LocationEvent`
-    attributes, the following attributes are defined:
+    In addition to the `Event` and `LocationEvent` attributes, the below event
+    attributes are defined.
```

Comment:
This still reads awkwardly.  Maybe?

```suggestion
    A MouseEvent has a number of special attributes in addition to those defined by the parent `Event` and `LocationEvent` classes:
```

Reply:
Yea, even though numpydoc says the order for classes is the same as normal functions with Parameters replaced by Attributes, the Attributes appear _after_ Examples. So I needed to tweak the wording, but it's still not great.

I'm not sure if it's numpydoc that's wrong, or if this is a theme bug.

Reply:
I reported this to numpydoc: numpy/numpydoc#331